changelog.md

```diff
@@ -1,11 +1,16 @@
 # Change Log
 This file contains all the notable changes done to the Ballerina NATS package through the releases.
 
-[Unreleased]
+## [2.9.1] - 2023-08-21
 
 ### Changed
-- [[#4237] Exit the Listener When Panic Occurred](https://github.com/ballerina-platform/ballerina-standard-library/issues/4237)
 
+- [[#4733] Changed disallowing service level annotations in the compiler plugin](https://github.com/ballerina-platform/ballerina-standard-library/issues/4733)
```

Comment:
Is this release version correct?

Reply:
What do you mean 😮
I just released https://github.com/ballerina-platform/module-ballerinax-nats/releases/tag/v2.9.1

Reply:
There are no records for the previous releases. That's what I meant. In a given branch, the changelog should include all the changes released from that branch. In this case, the release was done using another branch. So basically, on this branch, these are not released. It is incorrect to mention them as released.

Reply:
ooo didn't know 😮
Update 02b06108f836229bd064de928937cc9e2a7f7b66